pcap.c

```diff
@@ -1900,12 +1903,18 @@ pcap_parsesrcstr(const char *source, int *type, char *host, char *port,
 		return (0);
 	}
 
-	if (strcmp(scheme, "rpcap") == 0) {
+	int is_rpcap = 0;
+	if (strcmp(scheme, "rpcaps") == 0) {
+		is_rpcap = 1;
+		if (uses_ssl) *uses_ssl = 1;
+	}
+	is_rpcap |= strcmp(scheme, "rpcap") == 0;
```

Comment:
I might be inclined to do

    if (strcmp(scheme, "rpcaps") == 0) {
        is_rpcap = 1;
        if (uses_ssl)
            *uses_ssl = 1;
    } else if (strcmp(scheme, "rpcap") == 0)
        is_rpcap = 1;

as there's no need to check for "rpcap" if we already know it's "rpcaps".

Reply:
> as there's no need to check for "rpcap" if we already know it's "rpcaps".

Ah true, I though `|=` would shortcut if `is_rpcap` were already true, but of course not as it's a _logical_ or. Will do.